arch/RISCV/RISCVDisassembler.c

```diff
@@ -0,0 +1,246 @@
+//===-- RISCVDisassembler.cpp - Disassembler for RISCV --------------------===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+/* Capstone Disassembly Engine */
+/* By Rodrigo Cortes Porto <porto703@gmail.com>, 2018 */
+
+#ifdef CAPSTONE_HAS_RISCV
+
+#include <stdio.h>		// DEBUG
+#include <stdlib.h>
+#include <string.h>
+
+#include "../../cs_priv.h"
+#include "../../utils.h"
+
+#include "../../MCInst.h"
+#include "../../MCInstrDesc.h"
+#include "../../MCFixedLenDisassembler.h"
+#include "../../MCRegisterInfo.h"
+#include "../../MCDisassembler.h"
+#include "../../MathExtras.h"
+
+#define GET_SUBTARGETINFO_ENUM
+#include "RISCVGenSubtargetInfo.inc"
+
+static uint64_t getFeatureBits(int mode)
+{
+	// support everything
+	return (uint64_t) - 1;
+}
+
+// Forward declare these because the autogenerated code will reference them.
+// Definitions are further down.
+static DecodeStatus DecodeGPRRegisterClass(MCInst * Inst, uint64_t RegNo,
```

Comment:
the current coding style has no space between `*` and variable name (so this should be `MCInst *Inst`, but not `MCInst * Inst`. would be grateful if you can fix this issue (that i can see in many places).

sorry if all of this sounds too nitpicking, but we want as clean code as possible.

Reply:
no worries man. Good coding style is good for long-term maintainence.